src/planner.rs

```diff
@@ -171,13 +271,32 @@ impl<C: 'static> Planner<C> {
     }
 
     /// Runs a custom system.
+    ///
+    /// ## Deprecated
+    ///
+    /// This method is deprecated,
+    /// because it blocks until
+    /// all the other systems are finished.
+    ///
+    /// Please use [`Planner::add_system`]
+    /// instead.
+    ///
+    /// [`Planner::add_system`]: struct.Planner.html#method.add_system
+    #[deprecated(note = "Add a system instead")]
```

Comment:
Should we keep it? It does not really integreate well, especially as soon as we have these dependencies